api/app.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const bodyParser = require('body-parser');
+const envPath = __dirname + '/.env';
+require('dotenv').config({path: envPath});
```

Comment:
why do you need to provide a pth here? .env usuallu located right in fornt of entry point